composer.json

```diff
@@ -93,7 +93,6 @@
         "codeception/module-cli": "^2.0",
         "codeception/module-phpbrowser": "^3.0",
         "maglnet/composer-require-checker": "^4.5",
-        "phpunit/phpunit": "^9.5",
```

Comment:
Why was it removed?

Reply:
Because codeception is used, and if it were an explicit dependency, composer-require-checker would fail.